reactive/cuda.sh

```diff
@@ -220,16 +218,9 @@ function install_cuda() {
       return
     fi
 
-    apt-get update -qq
-    # apt-get upgrade -yqq
     # In any case remove nouveau driver
     apt-get remove -yqq --purge libdrm-nouveau*
     # Here we also need to blacklist nouveau
-    apt-get install -yqq --no-install-recommends \
```

Comment:
@tvansteenburgh I could maybe see it being critical when installing nvidia, openblas, or cuda, but here?  These packages are already on most (all?) units by default:

```
$ juju deploy ubuntu
$ juju ssh ubuntu/0
...
ubuntu@ip-172-31-19-218:~$ sudo apt-get install --no-install-recommends git curl wget build-essential
Reading package lists... Done
Building dependency tree
Reading state information... Done
build-essential is already the newest version (12.1ubuntu2).
build-essential set to manually installed.
curl is already the newest version (7.47.0-1ubuntu2.2).
git is already the newest version (1:2.7.4-0ubuntu1.1).
wget is already the newest version (1.17.1-1ubuntu1.2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.

ubuntu@ip-172-31-19-218:~$ sudo apt-get install git curl wget build-essential
Reading package lists... Done
Building dependency tree
Reading state information... Done
build-essential is already the newest version (12.1ubuntu2).
curl is already the newest version (7.47.0-1ubuntu2.2).
git is already the newest version (1:2.7.4-0ubuntu1.1).
wget is already the newest version (1.17.1-1ubuntu1.2).
0 upgraded, 0 newly installed, 0 to remove and 0 not upgraded.
```

Doesn't seem to make a difference whether i use `--no-install-recommends` or not.  It's a no-op either way.